### PR TITLE
Instructions for accessing embedded Longhorn dashboard and updated Volume Snapshot page

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
**Summary**
Resolves [#2838](https://github.com/harvester/harvester/issues/2838)

- Added reference to (https://github.com/harvester/harvester/issues/2838#issuecomment-1314355041) explaining the replication and health status of restoring a new volume from a volume snapshot can be viewed on the embedded Longhorn dashboard. Added new screenshots.
- Updated `volume-snapshot` page for formatting and style. Added some more context about creating a volume. 

**Next steps**

### CODE REVIEW

File: docs/volume/volume-snapshots.md [hunk 1]
```diff
@@ -8,33 +8,48 @@ keywords:
 Description: Take a snapshot for a volume from the Volume page.
 ---
 
-## Create Volume Snapshots
+After creating a volume, you can create a volume snapshot and use a volume snapshot to restore a new volume. Volume snapshots allow you to copy or restore a volume's configuraiton without creating a completely new volume.
```
[the remark this answers is not in the document]
Comment (vickyhella) [reply docs/volume/volume-snapshots.md:11]: Slightly updated the introduction according to the K8s doc [here](https://kubernetes.io/docs/concepts/storage/volume-snapshots/), and added more info about how snapshots can be helpful

File: docs/volume/volume-snapshots.md [hunk 2]
```diff
@@ -8,33 +8,48 @@ keywords:
...
+1. Choose the volume that you want to take a snapshot of and select **⋮ > Take Snapshot**.
+
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
+
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+1. You can view all VM snapshots and check the status of this operation from the **Backup & Snapshot > VM Snapshots** page. When the **ReadyToUse** status is set to **true**, the snapshot is complete.
```
Comment (vickyhella) [inline docs/volume/volume-snapshots.md:26]:
```suggestion
1. You can view all VM snapshots and check the status of this operation from the **Backup & Snapshot > VM Snapshots** page. When the **ReadyToUse** status becomes **true**, the snapshot is created successfully.
```

File: docs/volume/volume-snapshots.md [hunk 3]
```diff
@@ -8,33 +8,48 @@ keywords:
...
 
-1. Click the `⋮` button and select the `Take Snapshot` option.
-1. Configure the `Name` of the new image and click `Create`.
+:::note
 
-![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
-![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+A recurring snapshot is currently not supported and is tracked via [harvester/harvester#572](https://github.com/harvester/harvester/issues/572)
```
Comment (vickyhella) [inline docs/volume/volume-snapshots.md:30]:
```suggestion
A recurring snapshot is currently not supported and is tracked via [harvester/harvester#572](https://github.com/harvester/harvester/issues/572).
```

File: docs/volume/volume-snapshots.md [hunk 4]
```diff
@@ -8,33 +8,48 @@ keywords:
...
 ![restore-volume-snapshot-3](/img/v1.2/volume/restore-volume-snapshot-3.png)
+1. If the source volume is not an image volume, you can select a different **StorageClass**. You can not change the **StorageClass** if the source volume is an image volume.
 ![restore-volume-snapshot-4](/img/v1.2/volume/restore-volume-snapshot-4.png)
+1. Select **Create** to finish restoring a new volume.
+
+### View the replication and health status in Longhorn
+
+1. You can view the replication and health status for the new volume from Longhorn dashboard. The Longhorn dashboard is available directly in the Harvester dashboard, but you must first enable developer tools & features. Go to the **Preference** page.
+![restore-volume-snapshot-5](/img/v1.2/volume/restore-volume-snapshot-5.png)
+1. Select the **Enable Developer Tools & Features**.
```
Comment (vickyhella) [inline docs/volume/volume-snapshots.md:52]:
```suggestion
1. Check the **Enable Developer Tools & Features** box.
```

**Review by vickyhella — COMMENTED** [review]
Only some nits and the rest LGTM!

**Comment by LucasSaintarbor** [comment]
@futuretea may you review these instructions when you get the chance? Thanks!

File: docs/volume/volume-snapshots.md [hunk 5]
```diff
@@ -8,33 +8,48 @@ keywords:
...
+1. Choose the volume that you want to take a snapshot of and select **⋮ > Take Snapshot**.
+
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
+
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+1. You can view all VM snapshots and check the status of this operation from the **Backup & Snapshot > VM Snapshots** page. When the **ReadyToUse** status becomes **true**, the snapshot is created successfully.
```
Comment (futuretea) [inline docs/volume/volume-snapshots.md:26]: This description is about `VM Snapshots` but this page is about `Volume Snapshots`. The user cannot see volume snapshots from the `VM Snapshots` page.

File: docs/volume/volume-snapshots.md [hunk 6]
```diff
@@ -8,33 +8,48 @@ keywords:
...
 ![restore-volume-snapshot-2](/img/v1.2/volume/restore-volume-snapshot-2.png)
+1. Specify the **Name** of the new volume.
 ![restore-volume-snapshot-3](/img/v1.2/volume/restore-volume-snapshot-3.png)
+1. If the source volume is not an image volume, you can select a different **StorageClass**. You can not change the **StorageClass** if the source volume is an image volume.
 ![restore-volume-snapshot-4](/img/v1.2/volume/restore-volume-snapshot-4.png)
+1. Select **Create** to finish restoring a new volume.
+
+### View the replication and health status in Longhorn
+
+1. You can view the replication and health status for the new volume from the Longhorn dashboard. The Longhorn dashboard is available directly in the Harvester dashboard, but you must first enable developer tools & features. Go to the **Preference** page.
```
Comment (futuretea) [inline docs/volume/volume-snapshots.md:50]: the replication and health status for volumes are independent of volume snapshots. Prefer to add a new page.
And we should note that we only support using the embedded Longhorn dashboards for debugging and validation purposes.

Comment (LucasSaintarbor) [reply docs/volume/volume-snapshots.md]: @futuretea Removing this section and opening a PR to add a page covering how to view replication and health status for volumes via the embedded longhorn dashboard

**Review by LucasSaintarbor — COMMENTED** [review]
@futuretea removed instructions for accessing embedded Longhorn dashboard. Is it okay to merge?

File: docs/volume/volume-snapshots.md [hunk 7]
```diff
@@ -8,33 +8,39 @@ keywords:
...
+ ![create-volume-snapshot-1](/img/v1.2/volume/create-volume-snapshot-1.png)
+1. Enter a **Name** for the snapshot.
 
-After creating a volume, you can create volume snapshots by following the steps below:
+ ![create-volume-snapshot-2](/img/v1.2/volume/create-volume-snapshot-2.png)
+1. Select  **Create** to finish creating a new volume snapshot.
 
-1. Click the `⋮` button and select the `Take Snapshot` option.
-1. Configure the `Name` of the new image and click `Create`.
+1. You can check the status of this operation and view all volume snapshots from the **Backup & Snapshot > Volume Snapshots** page. When the **State** becomes **Active**, the volume snapshot is ready to use.
```
Comment (futuretea) [inline docs/volume/volume-snapshots.md:26]: You can check the status of this operation and view all volume snapshots belong to the volume from the volume's **Snapshots** tab. When the **Ready To Use** becomes **√**, the volume snapshot is ready to use.
image

---
Currently, the Active field cannot be used to determine whether a volumesnapshot is ready

Comment (futuretea) [reply docs/volume/volume-snapshots.md:26]: @LucasSaintarbor

Comment (LucasSaintarbor) [reply docs/volume/volume-snapshots.md]: Thanks @futuretea! Updated per [0a31124](https://github.com/harvester/docs/pull/302/commits/0a311248fd3533febdb8e17529850e91102e62e6)

**Review by futuretea — APPROVED** [review]
LGTM, thanks. Just need to squash commits

**Comment by LucasSaintarbor** [comment]
> LGTM, thanks. Just need to squash commits

Thanks! Adding these updates to v1.1 as well.

**Pull request merged**